ecommerce/config/EcommerceUiLabels.xml

```diff
@@ -350,6 +350,10 @@
         <value xml:lang="zh">分配明细</value>
         <value xml:lang="zh-TW">分配細項</value>
     </property>
+    <property key="EcommerceBestSelling">
```

Comment:
Hi @CharlesNereide, thanks for your contribution!

Can we change this key to "EcommercePopularCategories" instead, for consistency reasons?
You might notice this pattern in other labels too.

Reply:
At the beginning, that was my idea. But the template is called ShowBestSellingCategory.ftl, the screen is called showBestSellingCategory and the template is only for the category "Best Selling" (PCCT_BEST_SELL). So, for me, the label is dedicated to Best Selling category and it's better to spell the label according to its usage than to it spelling.
Moreover, I wanted change english label "Popular categories" by "Best selling". I think that "Popular categories" would be more efficient for the "Most popular" category (PCCT_MOST_POPULAR) for exemple.
But it's just my point of view ;).

Reply:
Thanks for the explanation, Charles!
I agree with your suggestion, updating the label to "Best Selling" would make more sense in reference to the context of the template.
Thanks!

Reply:
OK. I will propose a new PR from a new branch with more corrections on Best Selling template.

Reply:
You can close this one.
Thank you for your help.

Reply:
Hi @CharlesNereide, would you mind creating a ticket. It will help to track the effort effectively.
Thanks!